assets/src/scripts/components/hydrograph/legend.js

```diff
@@ -132,11 +147,14 @@ const legendDisplaySelector = createSelector(
     getCurrentVariableMedianMetadata,
     uniqueClassesSelector('current'),
     uniqueClassesSelector('compare'),
-    (showSeries, medianSeries, currentClasses, compareClasses) => {
+    waterwatchVisible,
+    getWaterwatchFloodLevels,
+    (showSeries, medianSeries, currentClasses, compareClasses, visible, getWW) => {
```

Comment:
nit. variable names shouldn't bet a verb. Can you replace getWW with floodLevels (or whatever noun you like)?

Reply:
Modified the variable name of getWW to be floodLevels.